llvm/test/CodeGen/DirectX/splitdouble_error.ll

```diff
@@ -0,0 +1,15 @@
+; RUN: opt -S -scalarizer -dxil-op-lower -mtriple=dxil-pc-shadermodel6.3-library %s 2>&1 | FileCheck %s
+
+; DXIL operation splitdouble doesn't support vector types.
+; XFAIL: * 
+
+define noundef <3 x i32> @test_vector_double_split(<3 x double> noundef %D) local_unnamed_addr {
```

Comment:
Since the scalarization pr merged before this pr this should no longer be an xfail. Consolidate this pr with  the existing split-double.ll test file.